Mathlib/Analysis/Normed/Group/Basic.lean

```diff
@@ -4,8 +4,10 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Patrick Massot, Johannes Hölzl, Yaël Dillies
 -/
 import Mathlib.Analysis.Normed.Group.Seminorm
+import Mathlib.Topology.Instances.ENNReal.Defs
```

Comment:
This import increase is needed to speak of a topological space structure on `ENNReal`.

Reply:
The only way I see to avoid this is to move these definitions to `Continuity`.

Reply:
Yael pointed out that this import was already added back in #20966 (for the same reason), and only removed in #21554 (which split that file). Hence, I would argue that keeping ENNReal out of `Basic.lean` only worked in the short-term, and this import is really needed.

Reply:
#22562 should make the situation much better. Out of interest, could you check how much imports increase if you rebase your PR on Anne's branch?

Reply:
I have just done so: let's wait what CI says.